proto/storage/compliance_operator_v2.proto

```diff
@@ -157,13 +157,11 @@ message ComplianceOperatorCheckResultV2 {
   map<string, string> labels = 9;
   map<string, string> annotations = 10;
   google.protobuf.Timestamp created_time = 11 [(gogoproto.moretags) = 'search:"Compliance Check Result Created Time,hidden"'];
-  string standard = 12 [(gogoproto.moretags) = 'search:"Compliance Standard,hidden"'];
-  string control = 13;
   string scan_name = 14 [(gogoproto.moretags) = 'search:"Compliance Scan Name,hidden" sql:"fk(ComplianceOperatorScanV2:scan_name),no-fk-constraint"'];
   string scan_config_name = 16 [(gogoproto.moretags) = 'search:"Compliance Scan Config Name" sql:"fk(ComplianceOperatorScanConfigurationV2:scan_config_name),no-fk-constraint"'];
   string rationale = 17;
-  repeated string valuesUsed = 18;
-  repeated string warnings = 19;
+  repeated string valuesUsed = 12;
+  repeated string warnings = 13;
```

Comment:
can you reorder this in tag order?

Reply:
🤣 yes, I thought about doing that while I was there, but laziness one.